### ISSUE TITLE
New line characters in help for a function argument

### ISSUE THREAD

**Issue by PGS62** [Excel-DNA/IntelliSense#121]
I'm using Excel-DNA Intellisense for VBA functions, which involves providing a worksheet named `_Intellisense_`. All works great, though I find that new line characters (ascii 10) in the "argument help" have no effect in the intellisense popup. I've tried using ascii 13 and the html tag `<br>` to no avail.

Is this something that could be fixed?

To illustrate, here's help for a [function](https://github.com/PGS62/VBA-CSV) that parses a .csv file to an array. In the image below, I show the intellisense that I'm able to achieve (on the left) versus what I would like to achieve (with line feeds, mocked up on the right).
![image](https://user-images.githubusercontent.com/18028484/198975753-080491ad-dc95-4960-824a-ffa5044d92a0.png)

**Comment by govert** [comment]
@PGS62 I see what is happening. When you edit a cell to add a new line (with Alt+Enter) then Excel puts only a newline (ASCII char 10) into the line. But the Excel-DNA IntelliSense is splitting the lines using the standard Windows newline sequence, with _**both**_ a carriage return and a newline (ASCII 13 & 10).
You can get string with these delimiters into the cell with a formula like
```
="My first line" & CHAR(13) & CHAR(10) & "and then the next line" & CHAR(13) & CHAR(10) & "and the rest"
```

It displays the same in the cell as the newline only, but now the Excel-DNA processing will pick up and display the extra lines.

It's probably worth updating the IntelliSense to also accept the newline characters alone.

**Comment by PGS62** [comment]
Yes, tweaking things so that newline in isolation works would be sensible. But many thanks for giving me an immediate solution!
